sdk/include/sdk/DataPointValue.h

```diff
@@ -194,10 +198,42 @@ template <typename T> class TypedDataPointValue : public DataPointValue {
         return DataPointValue::operator==(other) && m_value == other.m_value;
     }
 
+    [[nodiscard]] std::string getValueAsString() const override;
+
 private:
     T m_value;
 };
 
+template <typename T> std::string TypedDataPointValue<T>::getValueAsString() const {
+    // Return empty string in case of unspecific data types
+    return "";
```

Comment:
Better to throw an exception?!

Reply:
Did want to "keep it simple" for now - it's more a debug feature atm

Reply:
Still we should then at least print "unimplemented for: <type_id>", if possible